go/osagent/osagent.go

```diff
@@ -349,15 +349,15 @@ func LogicalVolumes(volumeName string, filterPattern string) ([]LogicalVolume, e
 		return nil, err
 	}
 
-	logicalVolumes := []LogicalVolume{}
+	var logicalVolumes []LogicalVolume
 	for _, lineTokens := range tokens {
 		logicalVolume := LogicalVolume{
 			Name:      lineTokens[1],
 			GroupName: lineTokens[2],
 			Path:      lineTokens[3],
 		}
 		logicalVolume.SnapshotPercent, err = strconv.ParseFloat(lineTokens[4], 32)
-		logicalVolume.IsSnapshot = (err == nil)
+		logicalVolume.IsSnapshot = err == nil
```

Comment:
I prefer it the previous way.
I assume some linter said it's not idiomatic or something? To me it's clearer the original way.